docs/docs/guides/how-to-run-autostaker-plugin.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 8
+---
+
+# How to run autostaker plugin
+
+The autostaker plugin assists an operator by automatically selecting
+sponsorships to stake in and to unstake from, and by adjusting existing
+stakes to reflect the earnings available from sponsorships relative to
+other sponsorships. It also makes sure the operator swiftly discovers
+and joins new sponsorships as they emerge. This frees the operator from
+having to manually monitor and adjust stakes.
+
+To run the autostaker plugin, a user must first be running one or more
+operator nodes. See [How to become an Operator](./become-an-operator.md)
+for more information.
+
+The autostaker plugin should be enabled on all of your nodes. E.g., say you
+are  running 4 nodes in total, each should be configured with autostaker
+enabled.
+
+The autostaker plugin is enabled by adding the following section to your node
+configuration (under the plugins section):
+```json
+{
+  "plugins": {
+    "autostaker": {
+      "operatorContractAddress": "<OPERATOR_ADDRESS>"
+    }
+  }
+}
+```
+
+In addition, you need to assign the staking agent (aka `CONTROLLER`) role for each nodes' wallet
+so they can adjust stakes on behalf of the operator. See [Staking angets](../streamr-network/network-roles/operators.md#staking-agents)
+for how this is done.
+
+:::info
+- When choosing to run the autostaker plugin, the user should not manually adjust
+their operator's stakes, e.g. via the Hub. The autostaker will most likely undo
+any manual changes by returning the stakes back to the state it considers optimal.
+- To withdraw earnings while the autostaker is enabled, you simply
+  initiate a withdraw and wait for the undelegation queue to be processed by the
+  autostaker. This will not be immediate.
```

Comment:
This will not be immediate, but will happen on the next activation of the autostaker (by default every X minutes).